msf_core/include/msf_core/gps_conversion.h

```diff
@@ -22,19 +22,34 @@
 
 namespace msf_core {
 class GPSConversion {
- private:
-  msf_core::Quaternion ecef_ref_orientation_;
-  msf_core::Vector3 ecef_ref_point_;
- public:
+public:
   EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+
   GPSConversion();
-  void InitReference(const double& latitude, const double& longitude,
-                     const double& altitude);
-  msf_core::Vector3 WGS84ToECEF(const double& latitude,
-                                const double& longitude,
-                                const double& altitude) const;
+  // Default copy constructor and assignment operator are OK.
+
+  ~GPSConversion();
```

Comment:
can you please reformat class / method names according to this:
https://github.com/ethz-asl/programming_guidelines/wiki/Cpp-Coding-Style-Guidelines#10-abbreviations-and-acronyms-must-not-be-uppercase-when-used-as-name-4

I think it's still not too late ;)